pii_team_frost.py

```diff
@@ -22,7 +22,11 @@ def find_amex(text) -> list:
 def find_us_ssn(text) -> list:
     """Finds all occurrences of a US social security number in a text string"""
     # match a 9 digit social security number
-    return []
+    SSNRegex = re.compile(r'\d\d\d-\d\d-\d\d\d\d')
```

Comment:
This works, however, \d\d\d can be shortened to \d{3} - similar for the other \d patterns you have